core/modules/layout/plugins/renderers/layout_renderer_editor.inc

```diff
@@ -115,9 +115,15 @@ class LayoutRendererEditor extends LayoutRendererStandard {
       'data-block-id' => $block->uuid,
     );
 
+    $block_admin_title = $block->getAdminTitle();
+    if (!$block->status) {
+      $attributes['class'][] = 'layout-editor-block-disabled';
+      $block_admin_title .= t(' (Disabled)');
```

Comment:
Translations should be given full context when possible, including a placeholder would be better than concatenating. 
```suggestion
      $block_admin_title = t('!title (Disabled)', array('!title' => $block_admin_title);
```

(The admin title is already escaped so `!title` is used instead of `@title`.)